node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -605,4 +606,10 @@ public void createSymlink(Path symlink, Path target) {
     protected Optional<CredentialsMaintainer> credentialsMaintainer() {
         return credentialsMaintainer;
     }
+
+    private static Duration warmUpDurationForZone(ZoneApi zone) {
+        return zone.getSystemName() == SystemName.cd || zone.getEnvironment().isTest()
+                ? Duration.ZERO
```

Comment:
Duration of zero still starts the container unconstrained, but then immediately attempts to suspend against orch. to set limits. Negative will start the container with the constraints https://github.com/vespa-engine/vespa/blob/44e2be1ea393b7556ab59e7f77a757d8d25be546/node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java#L215
```suggestion
                ? Duration.ofSeconds(-1)
```